### PR TITLE
chore: add animation to CopyToClipboard

### LINKED ISSUE #396: Update Copy to Clipboard
**User Story:**
As a round operator
I want feedback when I copy the application link to clipboard
So that I know that the copy was successful

**Acceptance Criteria**
GIVEN that I am on the Round Details page
WHEN I look at the "copy to clipboard" button
THEN I see the unselected state (lighter purple + "Copy to Clipboard" text)

GIVEN that I am on the Round Details page
WHEN I click on the "copy to clipboard" button
THEN the button changes to the selected state (darker purple + "Copied to Clipboard" text)
AND reverts to unselected state after a few seconds

**Product & Design Links:**
- [Unselected state](https://www.figma.com/file/K31Tf7tSn10BuN65VC6YLc/Round-Manager---Hi-Fidelity-Designs?node-id=1650%3A9835)
- [Selected state](https://www.figma.com/file/K31Tf7tSn10BuN65VC6YLc/Round-Manager---Hi-Fidelity-Designs?node-id=1655%3A9977)

**Tech Details:**

**Open Questions:**

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

Adds color transition on click for copy to clipboard animation

##### Refers/Fixes

fixes #396

##### Testing

Locally Tested

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Curious if there is a better / neater way of going about this

File: packages/round-manager/src/features/common/__tests__/CopyToClipboardButton.test.tsx [hunk 1]
```diff
@@ -12,27 +12,22 @@ Object.assign(navigator, {
...
   });
 
-  it("should display the CopyToClipboardButton with custom text ", () => {
-    renderWrapped(
-      <CopyToClipboardButton
-        textToCopy={textToCopy}
-        clipboardText={textToCopy}
-      />
-    );
+  it("should display Copied to clipboard when not clicked", () => {
```
Comment (shavinac) [inline packages/round-manager/src/features/common/__tests__/CopyToClipboardButton.test.tsx:25]:
```suggestion
  it("should display Copied to clipboard when clicked", () => {
```

File: packages/round-manager/src/features/common/CopyToClipboardButton.tsx [hunk 2]
```diff
@@ -1,24 +1,32 @@
...
     <Button
       type="button"
-      className={`inline-flex ${props.styles}`}
-      onClick={async () =>
-        await navigator.clipboard.writeText(props.textToCopy)
-      }
+      className={`inline-flex bg-violet-100 text-violet-600 ${active ? 'animate-[violetTransition_20s_ease-in]' : ''} ${props.styles}`}
+      onClick={async () => {
+        setActive(true);
+        setTimeout(() => setActive(false), 1000);
```
Comment (shavinac) [inline packages/round-manager/src/features/common/CopyToClipboardButton.tsx:22]: super nitpicky, but 1sec still feels too fast to me? would be good to ask Melissa to take a quick look

**Review by shavinac — APPROVED** [review]

File: packages/round-manager/src/features/common/CopyToClipboardButton.tsx (hunk 2 above)
Comment (thelostone-mc) [reply packages/round-manager/src/features/common/CopyToClipboardButton.tsx:22]: Fair ! I'll throw this up on staging and ask melissa to check this out

**Pull request merged**